contents/why_julia.md

```diff
@@ -152,19 +152,19 @@ This is known as the "Two-Language Problem" and we discuss next.
 ### The Two-Language Problem {#sec:two_language}
 
 The "Two-Language Problem" is a very typical situation in scientific computing where a researcher devises an algorithm or a solution to tackle a desired problem or analysis at hand.
-The solution is, then, prototyped in an easy to code language (like Python or R).
-If the prototype works, the researcher  would code in a fast language that would not be easy to prototype (C++ or FORTRAN).
-Thus, we have two languages involved in the process of of developing a new solution.
+The solution is then prototyped in an easy to code language (like Python or R).
```

Comment:
"then" is a non-essential part of the sentence and can, therefore, be removed. Formally, the comma's should be there, but I know that Jose also doesn't like them.

Middle way:
```suggestion
Then, the solution is prototyped in an easy to code language (like Python or R).
```